<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddKafkaConsumerForm</class>
 <widget class="QWidget" name="AddKafkaConsumerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kafka Consumer Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTableWidget" name="kafkaConsumerPropertiesTable">
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>0</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>Bootstrap servers</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Group id</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Topics</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Key</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>bootstrap.servers</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>localhost:9092</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEditable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>group.id</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>group_1</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEditable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>topics</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>test_1,test_2</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEditable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addPropertyRowButton">
       <property name="text">
        <string>Add propery</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="discardButton">
       <property name="text">
        <string>Discard</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
